django/contrib/gis/gdal/raster/band.py

```diff
@@ -222,8 +222,10 @@ def data(self, data=None, offset=None, size=None, shape=None, as_memoryview=Fals
             if as_memoryview:
                 return memoryview(data_array)
             elif numpy:
+                # reshape function need the size tuple with the height first
```

Comment:
Could reformulate to something like: "The numpy reshape function needs a reshape parameter with the height first."